diff --git a/node_modules/moti/src/skeleton/native.tsx b/node_modules/moti/src/skeleton/native.tsx
index 93d0bfc..57673ca 100644
--- a/node_modules/moti/src/skeleton/native.tsx
+++ b/node_modules/moti/src/skeleton/native.tsx
@@ -1,4 +1,5 @@
-import LinearGradient from 'react-native-linear-gradient'
+import { LinearGradient } from 'expo-linear-gradient'
+
 
 import SkeletonNative from './skeleton-new'
 import { MotiSkeletonProps } from './types'
